.github/workflows/testingWorkflow.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+on:
+  push:
+    branches: [dev,master]
+  pull_request:
+    branches: [dev,master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [15.x]
+        mongodb-version: ['6.0']
```

Comment:
Why is one version in single quotes while the other one is not? Does it works if you use `6.0` instead of `'6.0'`?